containers/vm-provisioning/config

```diff
@@ -0,0 +1,37 @@
+# this file is sourced by several scripts in this directory
+
+env=unknown
+if test -f instance; then
+    env=`cat instance`
+fi
+
+declare -A \
+    aws_image \
+    aws_instance_type \
+    aws_spot_max_price \
+
+# image=b8a0e7a5-fa58-4e2b-9fa5-e2c95159d3a2
+
+# Use `c9s` image from CentOS community.
+# https://www.centos.org/download/aws-images/
+# TODO: We should build a custom image from it by running `osh-worker-role`.
+# That would reduce the time to spin up new workers.
+aws_image[x86_64]=ami-0322b3bb1843afd17
+
+# We only support `x86_64` arch for OpenScanHub.
+# https://issues.redhat.com/browse/OSH-60
+# aws_image[aarch64]=ami-005c0bc715b4dab5a
+
+# TODO: Do we need this variable? Remove this?
+aws_spot_max_price[x86_64]=0.1
```

Comment:
@praiskup Could you comment on this?

Reply:
Are you using spot instances?  In Copr, we use on-demand instances, spot instances, and some reserved instances.  This variable was used by Copr to set up the "limit" price/hour/instance.

Reply:
As I understand, we only use on-demand instances.